scriptmodules/supplementary/esthemes.sh

```diff
@@ -130,6 +130,7 @@ function gui_esthemes() {
         'RetroHursty69 tributeSTrek'
         'RetroHursty69 tributeSWars'        
         'Saracade scv720'
+	'chicueloarcade Chicuelo'
```

Comment:
sorry - wrong indentation (we use 4 spaces for indentation). I will add it manually next time I do an edit.